### DISCUSSION TITLE
crossdb - can you publish mutiple .db files?

### DISCUSSION THREAD

**Discussion by justinkelly** [simonw/datasette#1536] [Q&A] [answered]
Hi Datasette crew

Is it possible to publish to a cloud host (like fly.io etc..) multiple .db files and use the cross database queries[1] feature to query across all of them?

1: https://simonwillison.net/2021/Feb/21/cross-database-queries/

Thanks for the amazing project

Cheers

Justin

**Comment by simonw** [comment, marked as answer]
Hi Justin,

That should be supported by both `datasette publish cloudrun`, `datasette publish heroku` and `datasette publish fly` (via the plugin) at the moment, using the `--extra-options` option:

    datasette publish cloudrun one.db two.db --service my-service --extra-options "--crossdb"

I can't recall if I've tested it against Fly but I've used it a bunch with Cloud Run and Heroku.

**Reply by justinkelly** [reply]
awesome - thats great news!!!! thanks so much for the crazy fast answer and the awesome project!!!

**Reply by huntiezz** [reply]
Cool
